src/coreclr/vm/corhost.cpp

```diff
@@ -359,10 +359,12 @@ HRESULT CorHost2::ExecuteAssembly(DWORD dwAppDomainId,
     UNINSTALL_UNWIND_AND_CONTINUE_HANDLER;
     UNINSTALL_UNHANDLED_MANAGED_EXCEPTION_TRAP;
 
+#if !defined(TARGET_BROWSER)
```

Comment:
This shouldn't be needed. The `LOG_EXECUTABLE_ALLOCATOR_STATISTICS` isn't normally set. How was this hit?